ras_party/support/session_decorator.py

```diff
@@ -1,10 +1,13 @@
 from functools import wraps
+import logging
 
 from flask import current_app
-from ras_common_utils.ras_error.ras_error import RasError, RasDatabaseError
-from structlog import get_logger
+import structlog
 
-log = get_logger()
+from ras_party.exceptions import RasError, RasDatabaseError
+
+
+logger = structlog.wrap_logger(logging.getLogger(__name__))
```

Comment:
In some files we're calling this logger and in others we're calling it log, should we be more consistent?

Reply:
Always. I thought I'd caught all the different ones.